servicedef/services_user.xml

```diff
@@ -0,0 +1,59 @@
+<services xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+          xsi:noNamespaceSchemaLocation="https://ofbiz.apache.org/dtds/services.xsd">
+    <description>Ofbiz-tutorial Services</description>
+    <vendor></vendor>
+    <version>1.0</version>
+    <service name="fetchAllUserData" engine="java" location="org.apache.ofbiz.ofbiztutorial.usermgr.services.UserMgrServices"
+             invoke="fetchAllUserData" auth="true" default-entity-name="UserAndParty">
+        <auto-attributes mode="IN" include="nonpk" optional="true"/>
```

Comment:
Why we didn't include PK here ?

Reply:
This section of the code is unnecessary. I previously wrote a service to retrieve data, but I now use a groovy script to fetch it directly. This section will be removed in the next commit.